packages/react/src/primitives/Menu/MenuItem.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { DropdownMenuItem } from '@radix-ui/react-dropdown-menu';
+
+import { ComponentClassNames } from '../shared/constants';
+import { MenuButton } from './MenuButton';
+import { MenuItemProps } from '../types';
+
+export const MENU_ITEM_TEST_ID = 'amplify-menu-item-test-id';
+
+export const MenuItem = React.forwardRef<
+  HTMLDivElement,
+  MenuItemProps & { children?: React.ReactNode }
```

Comment:
Why not just add `children` to `MenuItemProps`?

Reply:
Good point, updated.